templates/zerver/help/only-allow-admins-to-invite-new-users.md

```diff
@@ -0,0 +1,17 @@
+# Only allow admins to invite new users
+If you are an administrator, you can change a setting to only allow administrators
+to invite new users. This guide will teach you how to do this.
+
+1. Locate the cog
+(![cog](/static/images/help/cog.png)) on the upper right corner of the right
+sidebar and click on it to relase the dropdown menu. Select the **Administration**
```

Comment:
Typo in 'relase'. I think more appropriate would be 'click on it to show the menu'.